<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inscription_epreuve</class>
 <widget class="QDialog" name="inscription_epreuve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inscription à une épreuve</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_nom_ep_ins_ep">
           <property name="text">
            <string>Nom de l'épreuve :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combox_nom_ep_ins_ep"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_form_ins_ep">
           <property name="text">
            <string>Forme de l'épreuve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combox_forme_ins_ep"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_cat_ep_ins_ep">
           <property name="text">
            <string>Catégorie de l'épreuve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combox_cat_ep_ins_ep"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_date_ep_ins_ep">
           <property name="text">
            <string>Date de l'épreuve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combox_date_ep_ins_ep"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_num_ep_ins_ep">
           <property name="text">
            <string>Numéro de l'épreuve^</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="combox_num_ep_ins_ep"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="table_sp_ins_ep">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="validate_ins_ep">
       <property name="text">
        <string>Inscrire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_inscription_epreuve">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
